src/CRM/CivixBundle/Resources/views/Code/module.civix.php.php

```diff
@@ -355,8 +355,11 @@ function _<?php echo $mainFile ?>_civix_glob($pattern) {
  * Inserts a navigation menu item at a given place in the hierarchy.
  *
  * @param array $menu - menu hierarchy
- * @param string $path - path where insertion should happen (ie. Administer/System Settings)
- * @param array $item - menu you need to insert (parent/child attributes will be filled for you)
+ * @param string $path - path to where the item should be inserted. Made up of a
+ * series of 'name' attributes concatenated with /, e.g. 'Mailing',
+ * 'Administer/System Settings' or 'my_extension/submenu'
```

Comment:
FWIW, the pre-existing code-style here is a bit off, e.g. we're supposed to look more like:

```
 *  @param string $path
 *     Path to where the item should be inserted. Made up of a series of 'name' 
 *     attributes concatenated...
```

Aside: Perhaps there's a simpler word to describe this? Like "the parent path"?